src/helpers/parser/listItemParser.js

```diff
@@ -173,15 +173,17 @@ const parseTours = (data, skipLastDivider) => {
   }));
 };
 
-const parseCategories = (data, skipLastDivider) => {
+const parseCategories = (data, skipLastDivider, routeName = 'Category') => {
   return data?.map((category, index) => ({
     id: category.id,
     title: category.name,
     pointsOfInterestCount: category.pointsOfInterestCount,
     toursCount: category.toursCount,
-    routeName: 'Category',
+    routeName,
+    parent: category.parent ? true : false,
     params: {
       title: category.name,
+      categories: parseCategories(category.children, skipLastDivider, 'Index'),
```

Comment:
instead of hardcoding the route string here, `ScreenName.Index` could be used.

Reply:
change has been made